File: Unidad 2-Estructuras de Control/Equipo 6/C/Ej30.c
#include <stdio.h>
#include <stdlib.h>

/*
Autor: Alejandro Torre Reyes
Proceso: Calcular el Máximo Común Divisor entre 2 números enteros.
Entradas: 2 números enteros
Salidas: El MCD correspondiente
*/

int main()
{
    int a,b;
    printf("Ingrese dos numeros enteros");
    scanf("%d", &a);
    scanf("%d", &b);

    while(a != b){
        if(a>b){
            a = a-b;
        }
        else{
            b = b-a;
        }
    }
    printf("%d", a);
    return 0;
}

File: Unidad 2-Estructuras de Control/Ej30.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Autor: Alejandro Torre Reyes
Proceso: Calcular el Máximo Común Divisor entre 2 números enteros.
Entradas: 2 números enteros
Salidas: El MCD correspondiente
*/

namespace Ej30
{
    class Program
    {
        static void Main(string[] args)
        {
            int a, b;
            Console.WriteLine("Ingrese dos numeros enteros");
            a = Convert.ToInt32(Console.ReadLine());
            b = Convert.ToInt32(Console.ReadLine());

            while (a != b)
            {
                if (a > b)
                {
                    a = a - b;
                }
                else
                {
                    b = b - a;
                }
            }
            Console.WriteLine("MCD = {0}", a);
            Console.ReadKey();
        }
    }
}
